torch/csrc/jit/codegen/cuda/ops/normalization.cpp

```diff
@@ -23,7 +23,7 @@ TensorView* softmax(TensorView* x, int dim) {
   auto exp_val = exp(x_max_sub);
   auto sum_exp = sum(exp_val, {kReductionAxis});
   auto bcast_sum = broadcast(sum_exp, broadcast_mask);
-  auto y = div(exp_val, bcast_sum);
+  auto y = mul(exp_val, reciprocal(bcast_sum));
```

Comment:
Would it make sense to have this transformation somewhere during the lowering?

Reply:
I don't know if we enable this everywhere. I was trying to be a bit more conservative. We could just overwrite arith.cpp::div if we wanted.